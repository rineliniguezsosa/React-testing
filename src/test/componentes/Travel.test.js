import { render } from '@testing-library/react'

describe('pruebas en Travel.js', () => {
    test('debe de coincidir con el snapshot', () => {
        
    });
});