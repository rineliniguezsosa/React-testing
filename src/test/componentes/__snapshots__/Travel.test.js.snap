// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en Travel.js debe de coincidir con el snapshot 1`] = `
<div>
  <div
    class="card text-center m-1"
    style="width: 18rem;"
  >
    <i
      class="material-icons"
      style="font-size: 4rem;"
    >
      airplanemode_active
    </i>
    <h4>
      Travel Anywhere
    </h4>
    <p
      class="p-1"
    >
      Our premium package allows you to take exotic trips anywhere at the cheapest prices!
    </p>
  </div>
</div>
`;
